Pylint: Fix `logging-not-lazy` issues

## 💡 Description
Update the codebase to use "lazy" formatting for logging function calls. `pylint` report is attached (`pylint . --disable=all --enable=logging-not-lazy`):
```
************* Module pds.naif_pds4_bundler.utils.files
src/pds/naif_pds4_bundler/utils/files.py:89:12: W1201: Use lazy % or .format() formatting in logging functions (logging-not-lazy)
```

--- a/src/pds/naif_pds4_bundler/utils/files.py
+++ b/src/pds/naif_pds4_bundler/utils/files.py
@@ -86,11 +86,9 @@ def copy(src, dest):
         if e.errno == errno.ENOTDIR:
             shutil.copy(src, dest)
         else:
-            logging.warning(
-                f"-- Directory {src.split(os.sep)[-1]} not "
-                f"copied, probably because the increment "
-                "directory exists.\n Error: %s" % e
-            )
+            logging.warning("-- Directory %s not copied, probably because the "
+                            "increment directory exists.\n"
+                            "Error: %s", src.split(os.sep)[-1], e)
 
 
 def safe_make_directory(path):

